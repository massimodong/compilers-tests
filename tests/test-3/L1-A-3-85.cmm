float `Hf 
;
float W ( int _zQvZ6 [ 794601305 ] , struct QyKU8IhaK3fUPfW4rl 
  float DcqmCX3ADGdyVd [ 1862872522 ] , float bfEONa 
  , int gEmAHSaTPQ5o9CS ) {
}
float NpMuHDu19Ym9 , WHtKdGL [ 941807417 ] 1029188259.1571828413 
  , nEhdOOu3avIFuaVzW3I [ 1091090105 ] [ 1903108912 
  ] 


;
|| int || ;
!= 