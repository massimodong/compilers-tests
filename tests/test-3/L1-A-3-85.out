Error Type A at Line 1
Error Type B at Line 4
Error Type B at Line 7
Error Type B at Line 13
Error Type B at Line 13
Error Type B at Line 14
